src/Microsoft.Framework.Runtime/DependencyManagement/NuGetDependencyResolver.cs

```diff
@@ -222,11 +222,8 @@ public void Initialize(IEnumerable<LibraryDescription> packages, FrameworkName t
         {
             var defaultHashPath = defaultResolver.GetHashPath(package.Id, package.Version);
```

Comment:
This change isn't good enough. Take a look at the performance branch. It avoids any disk IO in the most common case.